api.swagger.yaml

```diff
@@ -141,6 +141,23 @@ paths:
       responses:
         '200':
           description: Successfully logged user out.
+  /pdf-export/close-job:
+    post:
+      tags:
+        - pdf export
+      description: PDF export render completion callback.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/pdf-export-completion-schema'
+      responses:
+        '200':
+          description: A matching PDF export job was found.
+        '400':
+          description: An incorrect or malformed token was provided.
```

Comment:
I think 401 might be more suitable here.

Reply:
Change applied.